docs/content/blog/cncf.md

```diff
@@ -2,7 +2,34 @@
 title: "Porter Has Joined the CNCF"
 description: "Porter has been accepted into the Cloud Native Computing Foundation (CNCF)"
 date: "2020-09-24"
-draft: true
 ---
 
-Porter has been accepted into the Cloud Native Computing Foundation (CNCF)...
+You may have noticed a flurry of activity recently on the Porter repositories
+and wondered what was afoot. It was all in preparation for the big reveal:
+Porter has been accepted into the Cloud Native Computing Foundation (CNCF)!<!--more-->
+
+When Jeremy Rickard and I created Porter less than two years ago, this is what
+we hoped for. We wanted to create a user-friendly tool for that would make Cloud
+Native Application Bundles (CNAB) accessible and deployments less stressful. It
+had to be user-first and community-first, without vendors getting in the way. So
+our sights were set on finding an independent home at the CNCF from day one. It
+means a lot to us to have built a community around Porter and tricked the CNCF
+into letting us in. 😉
+
+Porter is a CNCF [sandbox] project which is the first step for early stage
+projects where we can continue to grow and experiment. We have always encouraged
+contributions from everyone in the community, with a contribution ladder that
+allows anyone to become a maintainer, and we hope that the move to an
+independent foundation will attract more contributors, ideas and energy.
+
+Speaking of energy, [Ronan Flynn-Curran][ronan] is giving us a big boost with the new
+Porter logo. Meet Michelle the Porter Cat!
+
+![orange cat with a white belly standing and waving with its left arm, wearing a red bellhop hat and bowtie](/images/porter-logo.png)
+
+TODO: Say something about what we are doing next probably?
+
+TODO: Call to action! 🚀
+
```

Comment:
If you are interested in learning more about Porter, check out this introductory video. Learn more about the [Porter Community](https://porter.sh/community/) and contribute to the [project](https://github.com/getporter/porter/blob/main/CONTRIBUTING.md).

Reply:
In the near-term, we are focusing on [FILL IN BLANK FROM ROADMAP]. 

In addition to contributing ideas and energy to those initiatives, developers can contribute mixins using [skeletor](https://github.com/getporter/skeletor) as a starting point. Examples of mixins include the [Docker Mixin](https://github.com/getporter/docker-mixin) and [Terraform Mixin](https://github.com/getporter/terraform-mixin).